SECURITY.md

```diff
@@ -0,0 +1,22 @@
+# Security Policy
+
+To understand which version will receive security patches we recommend you to read the [Grails Support Schedule](https://grails.org/support-schedule.html).
+
+## Supported Versions
```

Comment:
I don't think we should put a table here.  We should link to the foundation support schedule and maintain that information in one place